### DISCUSSION TITLE
Possible tuning with DDT4ALL

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#1079] [General]
I have seen many adjustable methods for injectors, VVT solenoid/s...  Can anyone guide me to what can be done to improve performance of a engine? I have H4M engine in Dacia Dokker and it spits out 102HP that exact same engine in some Renault or Nissan models has 120HP (sure it may have some more sensors such as MAF... ) Im sure we can achieve at least 10%-15% increase of power with some adjustments...

**Comment by matzel82** [comment]
I want explain it short.

NO. It’s completely a other Software at the ECU.

**Comment by DavidIgor** [comment]
so we won't achieve anything with settings in ddt4all?

**Reply by matzel82** [reply]
Not more power. You have to use tuning tools…than it’s possible, maybe.

it’s all the same engine with 3 classes of power. Only other software, so I know that. The way to more power ? No idea…

**Reply by DavidIgor** [reply]
that is what bugs me... same engines but power limited by software (mostly).
